src/main/java/com/checkmarx/flow/service/EmailService.java

```diff
@@ -81,5 +90,65 @@ private String generateContent(Map<String, Object> ctx, String template) {
         context.setVariables(ctx);
         return templateEngine.process(template, context);
     }
+
+    public void sendScanSubmittedEmail(ScanRequest request) {
+        if (isEmailNotificationAllowed()) {
+            String scanSubmittedSubject = "Checkmarx Scan Submitted for ".concat(request.getNamespace()).concat("/").concat(request.getRepoName());
+            String scanSubmittedMessage = "Checkmarx Scan has been submitted for ".concat(request.getNamespace()).concat("/").concat(request.getRepoName())
```

Comment:
It's more readable with String.format().

Reply:
Changed to use String.format()